### PR TITLE
feat: add apps script integration

### PR DESCRIPTION
**Pull request by BobobUnicorn**

### CODE REVIEW

File: .clasp.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "scriptId": "1blO5oWqxF4kQ35PO5tj3lCYWn-8AKLv-mR3lJIzWp-LsIpP7EA46XEUY",
```
Comment (nullishamy) [inline .clasp.json:2]: Assuming this is ok to expose?

File: appsScript/index.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    method: 'post',
+    contentType: 'application/json',
+    payload: JSON.stringify(payload),
+    headers: {
+      Authentication: token
+    }
+  })
+}
+
+function createMenu (): void {
```
Comment (nullishamy) [inline appsScript/index.ts:142]: When does this run? This should not be visible to end users. Would be ideal to document this process.

File: .clasp.json (hunk 1 above)
Comment (nullishamy) [inline .clasp.json:2]: Assuming this is safe to expose?

**Review by nullishamy — CHANGES_REQUESTED** [review]

File: .clasp.json (hunk 1 above)
Comment (BobobUnicorn) [reply .clasp.json]: Can't do much with a script ID; Forms/Apps Script requires authentication in order to modify or view the deployed script.

File: appsScript/index.ts (hunk 2 above)
Comment (BobobUnicorn) [reply appsScript/index.ts:142]: This creates a menu that form editors can use to set up the form's integration.

**Review by nullishamy — APPROVED** [review]
LGTM

**Pull request merged**